openshift/template.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: fabric8-analytics-bigquery-manifests-job
+metadata:
+  name: fabric8-analytics-bigquery-manifests-job
+  annotations:
+    description: fabric8-analytics-bigquery-manifests-job
+objects:
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: fabric8-analytics-bigquery-manifests-job
+    annotations:
+      description: fabric8-analytics-bigquery-manifests-job
+  spec:
+    successfulJobsHistoryLimit: 4
+    failedJobsHistoryLimit: 0
+    concurrencyPolicy: "Forbid"
+    schedule: "${CRON_SCHEDULE}"
+    jobTemplate:
+      spec:
+        template:
+          spec: 
+            restartPolicy: Never
+            containers:
+            - name: bigquery-manifests
+              image: "djsarkar/bigquery-manifests:latest"
```

Comment:
@dipanjanS this is not the right image. Please provide the quay.io rhel image reference as that is going to be deployed.

Reply:
Done @samuzzal-choudhury